lib/XTR/ImageProcessingTask.php

```diff
@@ -75,19 +70,19 @@ private function runImageProcessing()
         $images = $this->mediaFactory->query(null, ['released' => 0, 'allModules' => 1, 'imageProcessing' => 1]);
 
         $libraryLocation = $this->config->getSetting('LIBRARY_LOCATION');
-        $resize_threshold = $this->config->getSetting('DEFAULT_RESIZE_THRESHOLD');
+        $resizeThreshold = $this->config->getSetting('DEFAULT_RESIZE_THRESHOLD');
 
         // Get list of Images
         foreach ($images as $media) {
 
             $filePath = $libraryLocation . $media->storedAs;
-            list($img_width, $img_height) = @getimagesize($filePath);
+            list($imgWidth, $imgHeight) = @getimagesize($filePath);
 
             // Orientation of the image
-            if ($img_width > $img_height) { // 'landscape';
-                $this->imageProcessingService->resizeImage($filePath, $resize_threshold, 1080);
+            if ($imgWidth > $imgHeight) { // 'landscape';
+                $this->imageProcessingService->resizeImage($filePath, $resizeThreshold, 1080);
```

Comment:
I didn't notice this during the last review, but it looks odd to be passing 1080 into here? if its necessary can we add a comment to explain why we do that?

It feels more natural to pass `null` into the parameter which we don't want to change?